gradle/libs.versions.toml

```diff
@@ -1,14 +1,16 @@
 [versions]
 awaitility = '4.2.0'
+caffeine = '3.1.1'
```

Comment:
is caffeine version not in the BOM?

Reply:
it's not in the BOM, it's a local dependency that's only used by `session` in core